packaging/packager_rpm.sh

```diff
@@ -5,40 +5,42 @@ readonly distro="${DISTRO:-fedora}"
 readonly distro_version="${VERSION:-30}"
 readonly distro_root="${ROOT:-$HOME}"
 
-# rpm dependencies
```

Comment:
The "rpm dependencies" is missing here but is present in the Debial variant of the packager script.

Reply:
How about we just remove it in both cases since it is clear the rpm packager is prioritizing RPM-s and the deb packager is prioritizing Debian packages.